autopush/logging.py

```diff
@@ -113,12 +113,17 @@ def __call__(self, event):
 
     def raven_log(self, event):
         f = event["log_failure"]
-        stack = extra = None
+        stack = None
```

Comment:
Forgot to remove the other "stack". Client info should have more than enough info to help us nail down bugs.